AuthorYYMMDD_fignr_001.yaml

```diff
@@ -0,0 +1,74 @@
+# This file can exist with the "source" information only, the entry data exists should be marked as "False" 
+# to indcate that neither the metadata nor the actual data were digitized or provided.
+# use "null" to idicate that a value is not available in the literature
+version: 1 # Is this necessary. Isn't this anyway given by git versioning
+
+source: 
+    doi: https://doi.org/10.1016/0039-6028(85)90985-9
+    bib id: Beden1985 # First Author and YYYY. Cross check for duplicats in the bib file.
+    figure: 2b # depending on the subfigure labels use 1, 1b, or 1_3
+    curve identifier: First cycle # use a unique short description to identify the curve in the figure
+    data exists: True
+    
+curator:
+    name: Albert Engstfeld
+    orcid: https://orcid.org/0000-0002-9686-3948
+    date digitized: 2021.06.01 # YYYY.MM.DD
+    
+data description:
+    type: d # d=digitized, o=original, r=raw

```

Comment:
I fear that when you type `digitized` it will easily lead to spelling errors. How about converting `d`, `o` and `r` into the spelled-out form during the conversion from `yaml` to `json`?

Reply:
```suggestion
    type: digitized # digitized, original, raw

```